src/shared/scrapers/VIR/index.js

```diff
@@ -0,0 +1,67 @@
+import * as fetch from '../../lib/fetch/index.js';
+import * as parse from '../../lib/parse.js';
+
+const scraper = {
+  country: 'VIR',
+  type: 'table',
+  url: 'https://doh.vi.gov/covid19usvi',
+  sources: [
+    {
+      url: 'https://doh.vi.gov',
+      name: 'United States Virgin Islands Department of Health'
+    }
+  ],
+  maintainers: [
+    {
+      name: 'Jacob McGowan',
+      github: 'jacobmcgowan'
+    }
+  ],
+  async scraper() {
+    const results = {
+      cases: 0,
+      tested: 0,
+      deaths: 0,
+      active: 0
+    };
+    const $ = await fetch.page(this.url);
+    const $widget = $('div.widget:has(div.title-box:has(h1:contains(COVID-19 Cases)))');
```

Comment:
Can you check to be sure you find the widget and throw an error if not? They may change the page.